server/services/store/sqlstore/migrate.go

```diff
@@ -68,7 +68,7 @@ func (s *SQLStore) getMigrationConnection() (*sql.DB, error) {
 	}
 	*settings.DriverName = s.dbType
 
-	db := sqlstore.SetupConnection("master", connectionString, &settings)
+	db, _ := sqlUtils.SetupConnection(s.logger, "master", connectionString, &settings, 5) // TODO: Need to find a way to get DBPingAttempts parameter from server
```

Comment:
Do we still need to get the DBPingAttempts configuration from the server?

Reply:
We can add the configuration for `dbPingAttemps` in `config.json` in focalboard server. This can do the job.